packages/@lwc/compiler/src/rollup-plugins/module-resolver.ts

```diff
@@ -45,14 +46,14 @@ function isFirstCharacterUppercased(importee: string) {
 
 function inferExtension(fileName: string, files: BundleFiles) {
     if (!path.extname(fileName)) {
-        const ext = VALID_EXTENSIONS.find(ext => files.hasOwnProperty(fileName + ext)) || '';
```

Comment:
on utilities like this, that are not subject to poisoning, we don't need to caching... but I'm fine either way.

Reply:
We can either go with this or disable the eslint rule https://eslint.org/docs/rules/no-prototype-builtins